src/auth/local.ts

```diff
@@ -0,0 +1,37 @@
+import { State } from "../index.js"
+import { createAccount } from "../lobby/index.js"
+import * as user from "../lobby/username.js"
+import * as storage from "../storage/index.js"
+import { USERNAME_STORAGE_KEY } from "../common/index.js"
+
+export const init = async (): Promise<State | null> => {
+  console.log("initialize local auth")
```

Comment:
We could switch out `console.log` with the debug logging functions, so they only print with `setup.debug({ enabled: true })`

Reply:
> setup.debug({ enabled: true })

This is also generally togglable in the console directly — no need for managing it ourselves

<img width="273" alt="Screen Shot 2021-12-09 at 10 01 27" src="https://user-images.githubusercontent.com/1052016/145420696-eacaa7dd-0ae5-4248-8c36-74dbeeaa755b.png">